packages/reactive-element/src/reactive-element.ts

```diff
@@ -1092,16 +1092,13 @@ export abstract class ReactiveElement
           }
         );
         if (shadowedProperties.length) {
-          issueWarning(
+          throw new Error(
```

Comment:
@aomarks what should we do for errors now that `issueWarning` takes a warning code?

Reply:
The error can just include the full URL; don't need to go through issueWarning, right?

Reply:
Yeah that seems fine, just put the code URL directly.

Reply:
Done